internal/bminventory/inventory.go

```diff
@@ -70,10 +70,14 @@ import (
 
 const DefaultUser = "kubeadmin"
 
-// 125 is the generic exit code for cases the error is in podman / docker and not the container we tried to run
-const ContainerAlreadyRunningExitCode = 125
 const WindowBetweenRequestsInSeconds = 10 * time.Second
 
+const (
```

Comment:
Should be in swagger

Reply:
(and agent should be updated sometime in the future to use the swagger enum too)

Reply:
Swagger does not support enums with name and numeric value or consts